add-on/panels/css/slideshow.css

```diff
@@ -0,0 +1,179 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans');
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+html {
+  font-size: 10px;
+  font-family: menu;
+  color: #fff;
+}
+
+body {
+  background: none;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  line-height: 2.4rem;  /* or 1.3rem original*/
+}
+.slideshow {
```

Comment:
Like I've just commented for the react code, please move this to a separate css file under add-on/panels/vendor as well, and update the license in a similar way.

Our specific css, e.g. html, body and everything under `/* slide-content */` I think can stay in this file.